src/pages/Account/Account.tsx

```diff
@@ -2,14 +2,11 @@ import "./Account.scss";
 import Button from "../../components/Button/Button";
 import { Link } from "react-router-dom";
 
-
 const Account = () => {
   return (
     <section className="account">
       <div className="account__info">
-        <h2 className="account__info--heading">
-          Way Of <br /> The Viking
-        </h2>
+        <h2 className="account__info--heading">Way Of The&nbsp;Viking</h2>
```

Comment:
This has caused the text to be moved onto one line, if this is accidental ignore this comment, otherwise please revert to the previous layout which matches the wireframes